docs/user-guide/configuring-http-api.rst

```diff
@@ -220,6 +221,16 @@ Your requests should have set a proper JWT header:
                      "Gl8eZFVfKXA6jhncgRn-I"
 
 
+The following is an example payload for a JWT token:
+
+.. code-block:: json
+
+      {
+          "user": {
+              "username": "<sender_id>",
+              "role": "user"
+          }
+      }
```

Comment:
Potentially dumb question, but where are they sending this payload?

Reply:
Only the Rasa Open Source server is processing it, and it only applies for endpoints enabled with `--enable-api`. No relationship with input channels.

Reply:
but does Rasa Open Source have an endpoint to generate a JWT token with this payload? Or do they need to create this elsewhere (since from what I see they need to create the JWT token to send as a header like in the block above)

Reply:
(e.g. Rasa X does have an endpoint)

Reply:
The person running the server must create the tokens themselves, and then make sure that whoever is making the request has included the token.

Reply:
@akelad Do you think this is ready to merge?

Reply:
sorry, just getting to this now. I think we should make it clear that people need to generate this themselves, externally from Rasa. Then we're good to merge